src/widget/form/chatform.cpp

```diff
@@ -783,9 +784,11 @@ void ChatForm::onScreenshotClicked()
 
 void ChatForm::doScreenshot()
 {
-    ScreenshotGrabber* screenshotGrabber = new ScreenshotGrabber(this);
-    connect(screenshotGrabber, &ScreenshotGrabber::screenshotTaken, this, &ChatForm::onScreenshotTaken);
+    screenshotGrabber = new ScreenshotGrabber(this);
+    connect(screenshotGrabber, &ScreenshotGrabber::screenshotTaken,
+            this, &ChatForm::onScreenshotTaken, Qt::QueuedConnection);
```

Comment:
Btw. by using a lambda you can avoid the member. ^^ Other than that - totally perfect.